test/test_transpose.py

```diff
@@ -35,8 +35,12 @@
 
 @unittest.skipUnless(BF_CUDA_ENABLED, "requires GPU support")
 class TransposeTest(unittest.TestCase):
+    first_test = True
+    
     def setUp(self):
-        bf.map_clear_cache()
+        if self.first_test:
```

Comment:
There's a `@classmethod setUpClass` that would probably be good for this, but it originates with python 3.2. https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUpClass

Reply:
All the great things that we will have when we stop supporting Python2.7...

Reply:
A frabjous day that shall be. Could start hash-tagging places to revisit once free, haha.